haproxy/haproxy.go

```diff
@@ -43,7 +43,7 @@ type HAproxy struct {
 
 // Constructs a properly configured HAProxy and returns a pointer to it
 func New(configFile string, pidFile string) *HAproxy {
-	reloadCmd := "haproxy -f " + configFile + " -p " + pidFile + " `[[ -f " + pidFile + " ]] && echo \"-sf $(cat " + pidFile + ")\"]]`"
+	reloadCmd := "haproxy -f " + configFile + " -p " + pidFile + " `[[ -f " + pidFile + " ]] && echo \"-sf $(cat " + pidFile + ")\"`"
```

Comment:
Maybe switch to fmt.sprintf() here?

Reply:
This is a copy-paste on code from upstream that I had to manually import. So I'm just leaving it like it is.